storages/parquet-storage/src/lib.rs

```diff
@@ -134,25 +146,25 @@ impl ParquetStorage {
         let mut key_counter: u64 = 0;
 
         if let Some(column_defs) = &fetched_schema.column_defs {
+            let primary_key_indices = fetched_schema.primary_key.as_ref().map(|primary_key| {
+                gluesql_core::executor::get_primary_key_column_indices(column_defs, primary_key)
+            });
```

Comment:
could we have this method in `data::Schema`? then we can use via `fetched_schema.get_primary_key_column_indices()`

Reply:
Reasonable, on it.

Reply:
Done